docs/development/releasing.md

```diff
@@ -56,8 +56,9 @@ will [trigger the release process](https://github.com/electron/electron/blob/796
 
 To monitor the build progress, see the following pages:
 
-- [208.52.191.140:8080/view/All/builds](http://208.52.191.140:8080/view/All/builds) for Mac and Windows
-- [jenkins.githubapp.com/label/chromium/](https://jenkins.githubapp.com/label/chromium/) for Linux
+- [208.52.191.140:8080/view/All/builds](http://208.52.191.140:8080/view/All/builds) for Mac
+- [https://circleci.com/gh/electron](https://circleci.com/gh/electron) for Linux
+- [https://windows-ci.electronjs.org/project/AppVeyor/electron] for Windows
```

Comment:
Markdown Break:
[https://windows-ci.electronjs.org/project/AppVeyor/electron] for Windows

Correct:
[windows-ci.electronjs.org/project/AppVeyor/electron](https://windows-ci.electronjs.org/project/AppVeyor/electron) for Windows